.github/workflows/reusable-build-test-release.yml

```diff
@@ -783,6 +789,39 @@ jobs:
           path: |
             *_markdown.txt
 
+  appinspect-api:
+    name: appinspect api ${{ matrix.tags }}
+    needs: build
+    if: |
```

Comment:
@mkolasinski-splunk do we want to run it always? or maybe only on pr and push to main?

Reply:
@artemrys yes, this is a good point.

Reply:
changed that